cves/CVE-2015-6826.yml

```diff
@@ -125,7 +142,12 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    While the file in question was under fairly active development by 
+    multiple developers during the three years between the VCC and fixes, the
+    function in question was not substantially updated, and there were no
+    breaking changes to the file as a whole. Therefore, the lifetime of the
+    vulnerability was rather uneventful.
```

Comment:
What about the project itself? 3 years is a long time. How did the project change?

Reply:
A good point that I had not considered. I'll do a little extra research and make an update here.

Reply:
Update added